docs/UserGuide/IoTDB-SQL-Language/DML-Data-Manipulation-Language.md

```diff
@@ -423,6 +423,43 @@ Total line number = 5
 It costs 0.014s
 ```
 
+#### Constant Timeseries Generating Functions
+
+The constant timeseries generating function is used to generate a timeseries in which the values of all data points are the same.
+
+The constant timeseries generating function accepts one or more timeseries inputs, and the timestamp set of the output data points is the union of the timestamp sets of the input timeseries.
+
+Currently, IoTDB supports the following constant timeseries generating functions:
+
+| Function Name | Required Attributes                                          | Output Series Data Type                      | Description                                                  |
+| ------------- | ------------------------------------------------------------ | -------------------------------------------- | ------------------------------------------------------------ |
+| CONST         | `value`: the value of the output data point <br />`type`: the type of the output data point, it can only be INT32 / INT64 / FLOAT / DOUBLE / BOOLEAN / TEXT | Determined by the required attribute  `type` | Output the user-specified constant timeseries according to the  attributes `value` and `type`. |
+| PI            | None                                                         | DOUBLE                                       | Data point value: a double value that is closer than any other to `pi`, the ratio of the circumference of a circle to its diameter. |
```

Comment:
This statement `a double value that is closer than any other to` is a little tricky and hard to understand. How about 
`a double value of `pi`, the ratio of the circumference of a circle to its diameter, with errors less than 1e-6`?
And I think we can use the letter π instead of `pi` in the description,  everyone may know this letter :)

Reply:
> a double value that is closer than any other to

Well, I copied the statement from the Java Standard Library... :D

I will modify the statement referring to yours, thanks for your suggestions!